apps/frontend/src/lib/backup/index.ts

```diff
@@ -473,3 +508,61 @@ export const createOAuthDeletionBackup = ({
     clientCreatedAt: new Date(),
   };
 };
+
+export interface CreateToggleSocialGraphBackupArgs {
+  email: string;
+  password: string;
+}
+
+// For backup of turning on or off
+export const createToggleSocialGraphBackup = ({
+    email,
+    password,
+  }: CreateToggleSocialGraphBackupArgs): CreateBackupData => {
+  const { authenticationTag, iv, encryptedData } = encryptBackupString({
+    backup: "",
+    email,
+    password,
+  });
+
+  return {
+    authenticationTag,
+    iv,
+    encryptedData,
+    backupEntryType: BackupEntryType.TOGGLE_SOCIAL_GRAPH,
+    clientCreatedAt: new Date(),
+  };
+};
+
+export interface CreateEdgeArgs {
+  email: string;
+  password: string;
+  edgeId: string;
+  timestamp: Date;
+}
+
+export const createEdgeBackup = ({
```

Comment:
don't we want to backup the username of the person this edgeId is for?

Reply:
I guess I don't see a downside of that and I do some future upsides (being able to label the global graph with your local "neighborhood"). Sure.

Reply:
Addressed (added username to local storage / backup)